src/api/goods.ts

```diff
@@ -1,11 +1,22 @@
-// eslint-disable-next-line
-const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
+const API_URL = 'https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json';
 
-export function getAll(): Promise<Good[]> {
+export const getAll: () => Promise<Good[]> = () => {
   return fetch(API_URL)
     .then(response => response.json());
-}
+};
 
-export const get5First = () => {};
+export const get5First = (): Promise<Good[]> => {
+  return fetch(API_URL)
+    .then(response => response.json())
+    .then(goods => goods.sort((firstGood: Good, secondGood: Good) => (
+      firstGood.name.localeCompare(secondGood.name)
+    )))
+    .then(sortedByAlphabet => [...sortedByAlphabet].slice(0, 5));
```

Comment:
I think it isn't  necessary to copy an array here, because the slice immutable method